### DISCUSSION TITLE
Printing on a virtual printer

### DISCUSSION THREAD

**Discussion by gigit-f** [sumatrapdfreader/sumatrapdf#2909] [General]
Hi to all.
Oftne I need to print a PDF file to a virtual printer, to add it too another. I use Win2Pdf but version 3.1.2 is the last that do this correctly. From 3.2 I cannot do it no more. I tried with 3.2, 3.3.3. and the last, 3.4.6
Can you tel me why?
Many thanks.

Gigi

**Comment by GitHubRulesOK** [comment]
No idea perhaps show your command line if using one or version/source of your win2pdf if I need to test that behavior via print dialogue but SumatraPDF is at the mercy of windows simple print processing so its the external drivers that should be tested for odd results 
using current SumatraPDF pre-rel and current win2pdf from its CHM I had no problem

![image](https://user-images.githubusercontent.com/12754913/187447625-2dfbda90-0bdf-48c9-8a5c-49990eec17af.png)

**Reply by gigit-f** [reply]
Thank you. I am using an old version of Win2Pdf but trying with other virtual printers (bullzip, microsoft print to pdf) it tuns.
I will consider to change someething.
